client/my-sites/backup/rewind-flow/download.tsx

```diff
@@ -141,7 +141,11 @@ const BackupDownloadFlow: FunctionComponent< Props > = ( {
 			<h3 className="rewind-flow__title">
 				{ translate( 'Currently creating a downloadable backup of your site' ) }
 			</h3>
-			<ProgressBar percent={ percent } />
+			<ProgressBar
+				isReady={ ! isDownloadURLNotReady }
+				percent={ percent }
+				initializationMessage={ translate( 'Initializing the download process' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Initializing the restore process' )` *ES Score: 7*